lab-doug/server.js

```diff
@@ -0,0 +1,57 @@
+const http = require('http');
+const parseUrl = require('url').parse;
+const parseQuery = require('querystring').parse;
+const cowsay = require('cowsay');
+const getContent = require(__dirname + '/lib/getContent');
+
+const port = process.argv[2] || 3000;
+
+const server = http.createServer(function(req, res){
+  //convert from string to object
+  req.url = parseUrl(req.url);
+  /*convert a query string to an object within the req.url object*/
+  req.url.query = parseQuery(req.url.query);
+
+  if(req.url.pathname == '/'){
```

Comment:
I'd probably always use '==='. Using just two is fine if you know what you're doing, but unless you can easily explain the exact differences between the two, it's best to just be in the habit of using three and staying safe.